funcs_test.sh

```diff
@@ -429,3 +509,13 @@ test_get_version_from_rc
 test_get_version_from_rc_with_extra
 test_exit_if_hotfix_ok
 test_exit_if_hotfix_not_ok
+
+test_deployment_cleanup_without_all_parameters
+test_deployment_cleanup_without_build_id
+test_deployment_cleanup_without_script_dir
+test_deployment_cleanup_without_app_bin
+test_deployment_cleanup_without_other_docker_build_file
+
+# WARNING: This will remove all the files except "scripts"
+# create a directory called "dummy-script" then copy the entire files inside it.
+# test_deployment_cleanup_with_id_build_id
```

Comment:
Situation like this @jarylolz12 kay sign na nga risky ang ato'ng script kay dili man siya puede ma-execute anywhere we want.